### ISSUE TITLE
Newbie installation issues

### ISSUE THREAD

**Issue by Griffin-Sullivan** [theforeman/theforeman-rel-eng#336]
I think it'll be easier to just run through this list and check them off or delete them as we fix.

- [ ] Tag_project needed me to assign a key for gpg git. Something like git config --global user.signingkey <my_key>
- [ ] Tag_push didn’t work because tool_belt used http clone had to reclone everything with ssh
- [ ] Bump_packaging needs foreman-packaging cloned into ~/dev and run the following:

```
Pip install obal
Yum install git-annex rpmdev-tools
In ~/.gitconfig
[alias]
  ph = "!git push $(whoami) HEAD --set-upstream --quiet"
  pr = "!hub pull-request"
  phr = "!git ph && git pr"
```
- [ ] Drop Append CLI release notes taken from the [hammer-cli](https://github.com/theforeman/hammer-cli/blob/3.6-stable/doc/release_notes.md) and [hammer-cli-foreman](https://github.com/theforeman/hammer-cli-foreman/blob/3.6-stable/doc/release_notes.md) changelogs, in [theforeman.org](https://github.com/theforeman/theforeman.org/trees/gh-pages/_includes/manuals/3.6/1.2_release_notes.md). From the instructions
- [ ] Add contributors script to theforeman-rel-eng
- [ ] Ctrl + d escapes script for release_announcement
- [ ] Specify that updates to the /web folder are done in master only
- [ ] Can’t push develop because of checks
- [ ] Can’t push directly to foreman-documentation master
- [ ] Update web/content/index.adoc and web/content/js/versions.js to add 3.9 as a release candidate” Shouldn’t exist
- [ ] Add RC tester role
- [ ] Close_redmine_version: Failed to require python-redmine
- [ ] Add “in a PR” to changing documentation to supported / unsupported
- [ ] Versions update should also add old release as unsupported
- [ ] Update full version before generating the release process 
- [ ] jenkins.JenkinsException: Could not parse JSON info for job when running jenkins related processes

**Comment by ekohl** [comment]
> * Tag_project needed me to assign a key for gpg git. Something like git config --global user.signingkey <my_key>

I don't think this is tracked before, but certainly something that belongs in the README

> * Tag_push didn’t work because tool_belt used http clone had to reclone everything with ssh

I've configured git to rewrite push URLs: `git config --global url.git@github.com:.pushInsteadOf https://github.com/`

> * Bump_packaging needs foreman-packaging cloned into ~/dev and run the following:

Note the `~/dev` directory is configurable:

https://github.com/theforeman/theforeman-rel-eng/blob/17a45ebb8a1aed4989e3884c473718c0b99ddeb4/settings#L57

You can override it in `settings.local`:

https://github.com/theforeman/theforeman-rel-eng/blob/17a45ebb8a1aed4989e3884c473718c0b99ddeb4/settings#L10-L13

It now clones foreman-packaging as needed, but https://github.com/theforeman/theforeman-rel-eng/commit/940a720b1f530153769d71575eecb54fc3b5a127 was probably merged after you got started.

Additional installations are added in https://github.com/theforeman/theforeman-rel-eng/pull/199

The additional aliases aren't documented. It may be better to avoid them and call the commands directly. Let's take that part to https://github.com/theforeman/theforeman-rel-eng/pull/337

**Comment by ekohl** [comment]
I've converted everything into separate issue because it's easier to have individual discussions.
